simln-lib/src/sim_node.rs

```diff
@@ -319,6 +319,7 @@ pub struct SimulatedChannel {
     short_channel_id: ShortChannelID,
     node_1: ChannelState,
     node_2: ChannelState,
+    pub exclude_capacity: bool,
```

Comment:
Another point for setting this in `new`, wouldn't need to make one field `pub`.

Also probably needs a comment to explain what we're excluding capacity from.